source/gui/guiHelper.py

```diff
@@ -458,3 +458,7 @@ class SIPABCMeta(wx.siplib.wrappertype, ABCMeta):
 	"""Meta class to be used for wx subclasses with abstract methods."""
 
 	pass
+
+
+def _stripAcceleratorFromLabel(label: str) -> str:
```

Comment:
Not currently used, though it will be if we re-add a failure state.

Reply:
If this is unused I think it should be removed from code.